### ISSUE TITLE
Setting datahub-gms KAFKA_CONSUMER_GROUP_ID breaks gms

### ISSUE THREAD

**Issue by dingobar** [linkedin/datahub#3133]
**Describe the bug**
It seems like setting the KAFKA_CONSUMER_GROUP_ID in GMS causes GMS to be unable to start.

**To Reproduce**
1. Install the prerequisites helm chart in a minikube cluster
2. Install the datahub helm chart
3. Observe that it works just fine
4. Set the KAFKA_CONSUMER_GROUP_ID in values.yaml and issue helm upgrade
5. Observe that GMS is not able to start (full stack trace below)

**Expected behavior**
GMS should start and all consumers should join the given consumer group ID.

**Screenshots**

**Desktop (please complete the following information):**
- Environment: Minikube
 - Version 0.8.10

**Additional context**
This is the error stack trace from the GMS logs:
```
09:09:50.870 [main] ERROR o.s.web.context.ContextLoader - Context initialization failed
java.lang.IllegalStateException: Another endpoint is already registered with id 'datadelivery-gexyo-datahub-dev'
        at org.springframework.util.Assert.state(Assert.java:73)
        at org.springframework.kafka.config.KafkaListenerEndpointRegistry.registerListenerContainer(KafkaListenerEndpointRegistry.java:170)
        at org.springframework.kafka.config.KafkaListenerEndpointRegistry.registerListenerContainer(KafkaListenerEndpointRegistry.java:146)
        at org.springframework.kafka.config.KafkaListenerEndpointRegistrar.registerAllEndpoints(KafkaListenerEndpointRegistrar.java:164)
        at org.springframework.kafka.config.KafkaListenerEndpointRegistrar.afterPropertiesSet(KafkaListenerEndpointRegistrar.java:158)
        at org.springframework.kafka.annotation.KafkaListenerAnnotationBeanPostProcessor.afterSingletonsInstantiated(KafkaListenerAnnotationBeanPostProcessor.java:263)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:896)
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
        at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:401)
        at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:292)
        at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:103)
        at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:921)
        at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:554)
        at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:888)
        at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:357)
        at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1443)
        at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1407)
        at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:821)
        at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:276)
        at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:106)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:106)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.server.Server.start(Server.java:407)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:106)
        at org.eclipse.jetty.server.Server.doStart(Server.java:371)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
        at org.eclipse.jetty.runner.Runner.run(Runner.java:520)
        at org.eclipse.jetty.runner.Runner.main(Runner.java:565)
2021-08-20 09:09:50.870:WARN:oejw.WebAppContext:main: Failed startup of context o.e.j.w.WebAppContext@3339ad8e{Open source GMS,/,file:///tmp/jetty-0_0_0_0-8080-war_war-_-any-3957490811295845490.dir/webapp/,UNAVAILABLE}{file:///datahub/datahub-gms/bin/war.war}
```

Here is the `values.yaml` file, which is just the quickstart values (without neo4j) with the environment variable set:
```yaml
# Values to start up datahub after starting up the datahub-prerequisites chart with "prerequisites" release name
# Copy this chart and change configuration as needed.
datahub-gms:
  enabled: true
  image:
    repository: linkedin/datahub-gms
    tag: "v0.8.10"
  extraEnvs:
    - name: KAFKA_CONSUMER_GROUP_ID
      value: "datadelivery-gexyo-datahub-dev"

datahub-frontend:
  enabled: true
  image:
    repository: linkedin/datahub-frontend-react
    tag: "v0.8.10"
  # Set up ingress to expose react front-end
  ingress:
    enabled: false

elasticsearchSetupJob:
  enabled: true
  image:
    repository: linkedin/datahub-elasticsearch-setup
    tag: "v0.8.10"

kafkaSetupJob:
  enabled: true
  image:
    repository: linkedin/datahub-kafka-setup
    tag: "v0.8.10"

mysqlSetupJob:
  enabled: true
  image:
    repository: acryldata/datahub-mysql-setup
    tag: "v0.8.10"

datahubUpgrade:
  enabled: true
  image:
    repository: acryldata/datahub-upgrade
    tag: "v0.8.10"

datahub-ingestion-cron:
  enabled: false

global:
  graph_service_impl: elasticsearch

  elasticsearch:
    host: "elasticsearch-master"
    port: "9200"

  kafka:
    bootstrap:
      server: "prerequisites-kafka:9092"
    zookeeper:
      server: "prerequisites-zookeeper:2181"
    schemaregistry:
      url: "http://prerequisites-cp-schema-registry:8081"

  sql:
    datasource:
      host: "prerequisites-mysql:3306"
      hostForMysqlClient: "prerequisites-mysql"
      port: "3306"
      url: "jdbc:mysql://prerequisites-mysql:3306/datahub?verifyServerCertificate=false&useSSL=true&useUnicode=yes&characterEncoding=UTF-8&enabledTLSProtocols=TLSv1.2"
      driver: "com.mysql.jdbc.Driver"
      username: "root"
      password:
        secretRef: mysql-secrets
        secretKey: mysql-root-password

  datahub:
    gms:
      port: "8080"
    mae_consumer:
      port: "9091"
    appVersion: "1.0"
```

Attached the full logs.
[logs.txt](https://github.com/linkedin/datahub/files/7020461/logs.txt)

**Comment by dexter-mh-lee** [comment]
Found this issue. MCE and MAE consumers are using the same name for the group id (This was okay before bc they were in separate pods). Will put out a fix asap

**Comment by dingobar** [comment]
This was fixed in 0.8.11
